clients/cli/src/main/java/org/apache/gravitino/cli/commands/Command.java

```diff
@@ -69,4 +79,19 @@ protected GravitinoAdminClient buildAdminClient() {
       return GravitinoAdminClient.builder(url).build();
     }
   }
+
+  /**
+   * Outputs the entity to the console.
+   *
+   * @param entity The entity to output.
+   */
+  protected <T> void output(T entity) {
+    if (outputFormat != null) {
+      if (outputFormat.equals(OUTPUT_FORMAT_TABLE)) {
+        TableFormat.output(entity);
+      } else if (outputFormat.equals(OUTPUT_FORMAT_PLAIN)) {
+        PlainFormat.output(entity);
+      }
+    }
```

Comment:
I think this logic can be improved/simplified.

Reply:
@justinmclean may have a different opinion. Could you please explain about it? https://github.com/apache/gravitino/pull/5606#discussion_r1859275498

Reply:
It should default to the plain output rather than output nothing. Also `--output plain` should be an option. `--output <unknown format>` should give an unsupported format message.

Reply:
Fixed in the commit: https://github.com/apache/gravitino/pull/5714/commits/b6b9b520cbd2e69be56d5e5f33fda2cb51a2c7f6